### PR TITLE
Create a n00b page for the public Hub

### PR DESCRIPTION
**Pull request by mbland**
Addresses #320. 18F/hub-pages-private#128 should go in first, and I need to add commits for the How-to page, but you can start reviewing this PR in the meanwhile.

cc: @wslack @bsweger @gboone @adelevie @jenniferthibault @konklone

### CODE REVIEW

**Comment by gboone** [comment]
I'm good with this. I _probably_ shouldn't be the decider, though.

**Comment by mbland** [comment]
OK, ready for review. Both internal and private versions look good on my machine.

@gboone Soon as we get these two PRs merged, can I get you to help me with a public Hub deployment?

**Comment by mbland** [comment]
Also cc: @brianwc

File: pages/how-to.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* GitHub
+* Google Apps and Tools
+* Legal and ethics questions
+* Freedom of Information Act
+* Federal records
+* Privacy
+* DevOps
+* Email groups and calendars
+* Washington D.C.
+* New York City
```
Comment (arowla) [inline pages/how-to.md]: Is there info about SF, too?

Comment (mbland) [inline pages/how-to.md]: Doesn't happen to have any at the moment.

**Pull request merged**